cdap-client-tests/src/test/java/io/cdap/cdap/client/QueryClientTest.java

```diff
@@ -64,11 +56,9 @@ public class QueryClientTest extends AbstractClientTest {
   private static final Logger LOG = LoggerFactory.getLogger(QueryClientTest.class);
 
   private ApplicationClient appClient;
-  private QueryClient queryClient;
   private NamespaceClient namespaceClient;
   private ProgramClient programClient;
   private ServiceClient serviceClient;
-  private ExploreClient exploreClient;
 
   @Override
   protected StandaloneTester getStandaloneTester() {
```

Comment:
If `QueryClient` has been removed, can the `QueryClientTest` also be removed?

Reply:
I was not sure of the functionality, that is why I didn't clean it up. If someone clarifies, I shall go ahead and remove it.

Reply:
Please remove